schedule/functional/extra_tests_gnome.yaml

```diff
@@ -26,6 +26,7 @@ conditional_schedule:
                 - x11/steam
                 - x11/wine
                 - x11/chrome
+                - python/qe-core/openbroadcastersoftware.py
```

Comment:
Let's go with the change you proposed for this `tests/qe-core/x11/openbroadcastersoftware` 

Is the `.py` intentional?

Reply:
Will do! `.py` is very intentional! By default we assume no extension means `.pm` files. So for python the extension is mandatory